runbotics/runbotics-orchestrator-ui/src/pages/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const IndexPage = () => {
+    return <div>Redirect to app/processes</div>;
```

Comment:
same here, whats index page by the way?

Reply:
Its there for redirecting users, so if someone hits runbotics.com/ they will be redirected to runbotics.com/app/processes. In the future there will be proxy to landing page